addons/hr_contract/data/hr_contract_demo.xml

```diff
@@ -1,10 +1,100 @@
+          <field name="structure_type_id" ref="hr_contract.structure_type_employee"/>
+          <field name="employee_id" ref="hr.employee_kar"/>
+          <field name="notes">This is Kate Richards's Part Time contract</field>
+          <field name="wage">2000.0</field>
+          <field name="state">open</field>
+          <field name="resource_calendar_id" ref="resource.resource_calendar_std"/>
+          <field name="hr_responsible_id" ref="base.user_admin"/>
+      </record>
+
+      <record id="hr_contract_frd_old" model="hr.contract">
+          <field name="name">Freya Dunkley Contract</field>
+          <field name="date_start" eval="(DateTime.now() + relativedelta(months=-6)).strftime('%Y-%m-%d')"/>
+          <field name="date_end" eval="(DateTime.now() + relativedelta(days=-14)).strftime('%Y-%m-%d')"/>
+          <field name="structure_type_id" ref="hr_contract.structure_type_employee"/>
+          <field name="employee_id" ref="hr.employee_frd"/>
+          <field name="notes">This is Freya Dunkley's contract</field>
+          <field name="wage">9000.0</field>
+          <field name="state">close</field>
+          <field name="resource_calendar_id" ref="resource.resource_calendar_std"/>
+          <field name="hr_responsible_id" ref="base.user_admin"/>
+      </record>
+
+      <record id="hr_contract_frd_new" model="hr.contract">
+          <field name="name">New Freya Dunkley Contract</field>
+          <field name="date_start" eval="(DateTime.now() + relativedelta(days=-13)).strftime('%Y-%m-%d')"/>
+          <field name="structure_type_id" ref="hr_contract.structure_type_employee"/>
+          <field name="employee_id" ref="hr.employee_frd"/>
+          <field name="notes">This is Freya Dunkley's Part Time contract</field>
+          <field name="wage">7000.0</field>
+          <field name="state">open</field>
+          <field name="resource_calendar_id" ref="resource_calendar_std_20h"/>
+          <field name="hr_responsible_id" ref="base.user_admin"/>
+      </record>
+
+      <record id="hr_contract_jog_old" model="hr.contract">
+          <field name="name">Sarah Davis Contract</field>
+          <field name="date_start" eval="(DateTime.now() + relativedelta(months=-10)).strftime('%Y-%m-%d')"/>
+          <field name="date_end" eval="(DateTime.now() + relativedelta(days=-10)).strftime('%Y-%m-%d')"/>
+          <field name="structure_type_id" ref="hr_contract.structure_type_employee"/>
+          <field name="employee_id" ref="hr.employee_sad"/>
+          <field name="notes">This is Sarah Davis's contract</field>
+          <field name="wage">3000.0</field>
+          <field name="state">close</field>
+          <field name="resource_calendar_id" ref="resource.resource_calendar_std"/>
+          <field name="hr_responsible_id" ref="base.user_admin"/>
+      </record>
+
+      <record id="hr_contract_sad_new" model="hr.contract">
+          <field name="name">New Sarah Davis Contract</field>
```

Comment:
Is it done on purpose that all 3 employees had a full-time, and now have a part time?
They could just have started with a part-time (or not) :shrug:

Reply:
Done! Make sarah's contract part time to full time